usmqe/usmscheduler.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf8 -*-
+"""
+Pytest plugin for scheduling tasks on nodes.
+.. moduleauthor:: Filip Balak <fbalak@redhat.com>
+"""
+import usmqe.usmssh as usmssh
+import datetime
+import random
+import string
+
+
+class Scheduler(object):
+    """
+    Wrapper around `usmssh` module that handles scheduling tasks on multiple
+    nodes simultaneously.
+    """
+
+    def __init__(self, nodes):
+        """
+        Args:
+            nodes (list): list of nodes used for command scheduling
+        """
+        self.nodes = nodes
+        self.ssh = usmssh.get_ssh()
```

Comment:
Would it make sense to allow to pass ssh object in constructor, but with None as a default, so that if is None, `usmssh.get_ssh()` is called to get it.

Reply:
I am not sure in which use case this would be used because object returned by `usmssh.get_ssh()` provides connections to ssh on other nodes but doesn't contain any information about these nodes (with each call to this object you have to provide node address). This method also returns the same global variable that would be sent in parameter that you propose (as seen in https://github.com/usmqe/usmqe-tests/blob/master/usmqe/usmssh.py#L34).

Reply:
Ah, ok. Never mind. This would be useful for (unit) testing only then.